jans-cedarling/cedarling/src/jwt/decoding_strategy/key_service.rs

```diff
@@ -77,7 +85,7 @@ impl KeyService {
             if let Some(key) = self.get_key_from_iss(&iss, &kid) {
                 return Ok(key.clone());
             } else {
-                println!("could not find {}, updating jwks", kid);
+                eprintln!("could not find {}, updating jwks", kid);
```

Comment:
Seems to me this should end up in a log somewhere, rather than on stderr.

Perhaps that change should be deferred until after this PR is accepted.

Reply:
i added a "TODO" string somewhere in PR #9939 about that... sorry it's such a mess right now, i'm planning to do a refactor which fixes a lot of the issues once the other PRs merges